src/launchpads/MK3/LaunchpadMK3.ts

```diff
@@ -0,0 +1,242 @@
+import midi from 'midi';
+import { findDevice, onExit } from '../../utils.js';
+import BaseLaunchpad from '../BaseLaunchpad.js';
+
+export type LaunchpadMK3Options = {
+  deviceName: RegExp,
+  debug: boolean,
+  xyMode: boolean,
+};
```

Comment:
This should probably be extracted into a `GenericLaunchpadOptions` or something similar

Reply:
Yeah.

Why don't we always have `xyMode` on? Isn't it just generally nicer to work with?

Reply:
That was mainly because I prefer it off for the software I wrote this for